lang/lexer/dialect_soussou.go

```diff
@@ -0,0 +1,75 @@
+package lexer
+
+import "regexp"
+
+type Soussou struct{}
+
+var _ Dialect = (*Soussou)(nil)
+
+func (m Soussou) Patterns() []RegexpPattern {
+	return []RegexpPattern{
+		{regexp.MustCompile(`\s+`), skipHandler},
+		{regexp.MustCompile(`dialect`), defaultHandler(DialectDeclaration, "dialect")},
+		{regexp.MustCompile(`konti`), defaultHandler(TypeInt, "konti")},
+		{regexp.MustCompile(`sèbèli`), defaultHandler(TypeString, "sèbèli")},
+		{regexp.MustCompile(`xamuara`), defaultHandler(KeywordElse, "xamuara")},
+		{regexp.MustCompile(`xa`), defaultHandler(KeywordIf, "xa")},
+		{regexp.MustCompile(`masen`), defaultHandler(Print, "masen")},
+		{regexp.MustCompile(`structure`), defaultHandler(Struct, "structure")},
+		{regexp.MustCompile(`labali`), defaultHandler(Main, "labali")},
+		{regexp.MustCompile(`fonction`), defaultHandler(Function, "fonction")},
+		{regexp.MustCompile(`ragbilen`), defaultHandler(Return, "ragbilen")},
```

Comment:
```suggestion
		{regexp.MustCompile(`fi`), defaultHandler(Return, "fi")},
```
I found that `fi` should be use for et return statement as it short and it mean `give`